frontend/src/modules/entitydetails/components/EntityDetails.js

```diff
@@ -168,6 +169,17 @@ export class EntityDetailsBase extends React.Component<InternalProps, State> {
         dispatch(machinery.actions.get(source, locale, pk));
     }
 
+    copyLinkToClipboard = () => {
+        const { locale, project, resource, entity } = this.props.parameters;
+
+        let string_link = `/${locale}/${project}/${resource}/?string=${entity}`
```

Comment:
This is just a relative link, without the `https://example.com/` part, so it won't work when pasted.